Hardware/CPU/AMD10CPU.cs

```diff
@@ -162,6 +180,22 @@ public AMD10CPU(int processorIndex, CPUID[][] cpuid, ISettings settings)
         }
       }
 
+      uint addr = Ring0.GetPciAddress(0, 20, 0);
+      if(Ring0.ReadPciConfig(addr, 0, out uint dev)) {        
+        Ring0.ReadPciConfig(addr, 8, out uint rev);
+
+        if(dev == 0x43851002)
+          cStatesIoOffset = (byte)((rev & 0xFF) < 0x40 ? 0xB3 : 0x9C);
+        else if(dev == 0x780B1022 || dev == 0x790B1022)
+          cStatesIoOffset = (byte)0x9C;
+      }
+      if(cStatesIoOffset != 0) {
+        cStatesResidency = new Sensor[]{
+          new Sensor("CPU Pkg C2", 0, SensorType.Level, this, settings),
+          new Sensor("CPU Pkg C3", 1, SensorType.Level, this, settings) };

```

Comment:
Can you name these as CPU Package (similar to Intel). So CPU Package C2, CPU Package C3.